dotCMS/src/main/java/com/dotmarketing/util/ImportUtil.java

```diff
@@ -1519,6 +1482,70 @@ can manage multilingual inserts for already stored records but not for the case
         }
     }
 
+    /**
+     * Get the site and folder from the given identifier or name
+     * @param idOrName the identifier or name
+     *                 it can be a host identifier, a folder identifier, a path to a folder or a host name
+     * @param user current user
+     * @return a pair with the host and folder
+     */
+    private static Pair<Host, Folder> getSiteAndFolderFromIdOrName(String idOrName, User user)
+            throws DotDataException, DotSecurityException {
+
+        if (UtilMethods.isSet(idOrName)) {
+
+            // Verify if the value belongs to a host identifier
+            Folder folder = null;
+            Host host = hostAPI.find(idOrName, user, false);
+
+            // If a host was not found using the given value (identifier) it can be a folder
+            if (!UtilMethods.isSet(host) || !UtilMethods.isSet(host.getIdentifier())) {
+                folder = folderAPI.find(idOrName, user, false);
+            }
+
+            // If a host or folder was not found using the given value (identifier)
+            // it can be a path to a folder or a host name
+            if ((!UtilMethods.isSet(folder) || !InodeUtils.isSet(folder.getInode()))
+                    && (!UtilMethods.isSet(host) || !UtilMethods.isSet(host.getIdentifier()))) {
+                if (idOrName.contains("//")) {
+                    // double slash is used to include the host name followed by the path
+                    String hostName = null;
+                    final String[] arr = idOrName.split("/");
```

Comment:
I think the StringPool has a constants for "/" also you can add a new one for host indicator

Reply:
Changed to constant from SlashPool